mapserver.h

```diff
@@ -1517,6 +1537,11 @@ The :ref:`REFERENCE <reference>` object
   /*                              legendObj                               */
   /************************************************************************/
 
+  #define MS_LEGEND_KEYSIZE_MIN 5
```

Comment:
Docs to update at https://mapserver.org/mapfile/legend.html

Reply:
Yes, once values are confirmed.